src/index.js

```diff
@@ -102,6 +102,9 @@ const unexpectedEnzyme = {
                   .text('}');
               }
               break;
+            case 'function':
+              startTag.text('={...}');
```

Comment:
`functionProp={λ}` ;-)

I'm in a conversation with Andreas about skipping the body for the function inspection, so that might also be a solution, but it would still be a bit noisy.